WordPress/src/main/java/org/wordpress/android/ui/reader/adapters/ReaderPostAdapter.java

```diff
@@ -309,15 +312,26 @@ private void toggleFollowButton(
 
         final boolean isAskingToFollow = !ReaderTagTable.isFollowedTagName(currentTag.getTagSlug());
 
+        final String slugForTracking = currentTag.getTagSlug();
+
         ReaderActions.ActionListener listener = succeeded -> {
             if (!succeeded) {
                 int errResId = isAskingToFollow ? R.string.reader_toast_err_add_tag
                         : R.string.reader_toast_err_remove_tag;
                 ToastUtils.showToast(context, errResId);
+            } else {
+                if (isAskingToFollow) {
```

Comment:
Just to double check: Do we really want to track whether the action succeeds/fails and not the button clicks? We usually track user interaction with the app (touch events) not results of actions. However, if it's in sync with iOS, we can keep it as is. Wdyt?

Reply:
>  However, if it's in sync with iOS, we can keep it as is. Wdyt?

@malinajirka - iOS confirmed it's tracked on the success of the operation. All is ready to go!